packages/material-ui/src/utils/integerPropType.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { expect } from 'chai';
+import { integerPropType } from '.';
+
+function callValidator(
+  validator,
+  { props },
+  propName = '',
+  componentName = '',
+  location = '',
+  propFullName = '',
+) {
+  return validator(props, propName, componentName, location, propFullName);
+}
+
+describe('integer', () => {
+  function assertPasses(validator, element, propName) {
+    expect(callValidator(validator, element, propName, 'integer test')).to.equal(null);
+  }
+
+  function assertFails(validator, element, propName) {
+    expect(callValidator(validator, element, propName, 'integer test')).to.be.instanceOf(Error);
+  }
+
+  it('Passes on undefined but failes on null value', () => {
```

Comment:
please keep the current naming convention, lowercase:
```suggestion
  it('passes on undefined but failes on null value', () => {
```

x the other cases